widgetastic_manageiq.py

```diff
@@ -808,8 +809,7 @@ class PaginationPane(View):
 
     @property
     def exists(self):
-        cur_view = self.browser.element(self)
-        return False not in self.browser.classes(cur_view)
+        return self.is_displayed
```

Comment:
awesome , I have missed this obvious way. Thanks.
I'd add some smoke tests for every widget if more things were converted to widgetastic.